src/components/layout.js

```diff
@@ -1,5 +1,4 @@
 import 'loaders.css';
-import PropTypes from 'prop-types';
```

Comment:
My old comment on keeping this as-is stands.

Reply:
```children: PropTypes.node.isRequired,``` is redundant - so nothing is left. I let this import to be here?

Reply:
As I've said before - I do want to have proper proptype definitions in the long run